From 89c8e84c6ab3b85ca37a1ee3a0731815073ccbf7 Mon Sep 17 00:00:00 2001
From: "Tao, Jing" <jing.tao@intel.com>
Date: Thu, 23 Oct 2014 12:33:14 +0800
Subject: [PATCH 107/117] mt9m114-ecs7: delete 480p output sensor settings

The 720x480 sensor settings is not FULL FOV which is not useful
and will cause narrow FOV in 640x480 and 320x240 capture as it will
be selected.

Delete this settings and use ISP's scaling based on other Full FOV
sensor settings.

Change-Id: I9f17e08f33397e485a3c7ffa61d2cb712cad2137
Tracked-On: https://bz01p-vied.ir.intel.com/show_bug.cgi?id=4326
Signed-off-by: Tao, Jing <jing.tao@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/25151
Reviewed-by: Tian Shu Qiu <tian.shu.qiu@intel.com>
Reviewed-by: Bingbu Cao <bingbu.cao@intel.com>
Build: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
Maintainer: Wei Tao <wei.tao@intel.com>
---
 .../external_drivers/camera/drivers/media/i2c/mt9m114.c |  9 ---------
 .../external_drivers/camera/drivers/media/i2c/mt9m114.h | 17 -----------------
 2 files changed, 26 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c b/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
index 6a3136b..9fc75e5 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
@@ -750,10 +750,6 @@ static int mt9m114_res2size(unsigned int res, int *h_size, int *v_size)
 		vsize = MT9M114_RES_720P_SIZE_V;
 		break;
 #else
-	case MT9M114_RES_720_480p_768:
-		hsize = MT9M114_RES_720_480p_768_SIZE_H;
-		vsize = MT9M114_RES_720_480p_768_SIZE_V;
-		break;
 	case MT9M114_RES_736P:
 		hsize = MT9M114_RES_736P_SIZE_H;
 		vsize = MT9M114_RES_736P_SIZE_V;
@@ -960,11 +956,6 @@ static int mt9m114_set_mbus_fmt(struct v4l2_subdev *sd,
 				MISENSOR_R_MODE_MASK, MISENSOR_NORMAL_SET);
 		break;
 #else
-	case MT9M114_RES_720_480p_768:
-		ret = mt9m114_write_reg_array(c, mt9m114_720_480P_init, NO_POLLING);
-		ret += misensor_rmw_reg(c, MISENSOR_16BIT, MISENSOR_READ_MODE,
-				MISENSOR_R_MODE_MASK, MISENSOR_NORMAL_SET);
-		break;
 	case MT9M114_RES_736P:
 		ret = mt9m114_write_reg_array(c, mt9m114_736P_init, NO_POLLING);
 		ret += misensor_rmw_reg(c, MISENSOR_16BIT, MISENSOR_READ_MODE,
diff --git a/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.h b/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.h
index 6e2183d..58322d4 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.h
@@ -179,7 +179,6 @@ enum {
 	MT9M114_RES_576P,
 	MT9M114_RES_720P,
 #else
-	MT9M114_RES_720_480p_768,
 	MT9M114_RES_736P,
 	MT9M114_RES_864P,
 #endif
@@ -437,22 +436,6 @@ static struct mt9m114_res_struct mt9m114_res[] = {
 	},
 #else
 	{
-	.desc	= "480P",
-	.res	= MT9M114_RES_720_480p_768,
-	.width	= 736,
-	.height = 496,
-	.fps	= 30,
-	.used	= 0,
-	.regs	= NULL,
-	.skip_frames = 1,
-
-	.pixels_per_line = 0x0644,
-	.lines_per_frame = 0x03E5,
-	.bin_factor_x = 1,
-	.bin_factor_y = 1,
-	.bin_mode = 0,
-	},
-	{
 	.desc	= "720P",
 	.res	= MT9M114_RES_736P,
 	.width	= 1296,
-- 
1.9.3

